flake.nix

```diff
@@ -0,0 +1,132 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+  };
+
+  outputs = inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      # TODO(XXX): in theory this flake could support aarch64-linux,
+      #   x86_64-darwin and aarch64-darwin, but it is untested.
+      systems = [ "x86_64-linux" ];
+      perSystem = { config, self', pkgs, lib, system, ... }:
+        let
+          buildToolsVersion = "30.0.3";
+          platformVersion = "33";
+          rustTargets = [ "x86_64-linux-android" ];
+
+          devDeps = with pkgs; [
```

Comment:
Maybe add [`der-ascii`](https://search.nixos.org/packages?channel=23.11&show=der-ascii&from=0&size=50&sort=relevance&type=packages&query=der-ascii) here too since its packaged and up-to-date?

Reply:
Sure, good idea :+1: 

Added.